org.openmsx.openMSX.metainfo.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2001-2020 openMSX team -->
+<component type="desktop">
+	<id>org.openmsx.openMSX</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-2.0</project_license>
+	<name>openMSX</name>
+	<summary>Emulate all espects of the MSX with high accuracy</summary>
+	<categories>
+		<category>Emulation</category>
+	</categories>
+	<releases>
+		<release version="16.0" date="2020-08-26"/>
+	</releases>
+	<description>
+		<p>
+			openMSX is an emulator for the MSX home computer system. MSX is an old
+			Z80-based family of home computers as an attempt to establish
+			a single standard in home computing similar to VHS in video.
+		</p>
+		<p>
+			The MSX standard has been designed by a company called ASCII in cooperation
+			with Microsoft which has provided a firmware version of its extended BASIC
+			(called "MicroSoft eXtended BASIC") for the machine, which explains the
+			MSX name.
+		</p>
+		<p>
+			In addition to emulating MSX, MSX2, MSX2+, MSX Turbo R and many of it's
+			peripherals, openMSX also supports emulating the ColecoVision game console
+			and the SpectraVideo SVI-318 and SVI-328 home computer systems.
+		</p>
+		<p>
+			Note: by default the openMSX emulator only ships with the C-BIOS open-source
+			BIOS implementation. This BIOS is restricted to cartridge based MSX games. For
+			a full MSX or SVI emulation it is necessary to provide your own MSX and/or SVI
+			BIOS ROMs, which must be placed in the ~/.openMSX/share/systemroms directory.
+		</p>
+	</description>
+	<url type="homepage">https://openmsx.org</url>
+	<url type="bugtracker">https://github.com/openMSX/openMSX/issues</url>
+	<url type="help">https://openmsx.org/manual/user.html</url>
+	<screenshots>
+		<screenshot type="default">https://openmsx.org/images/screenshots/mlimit3.png</screenshot>
+		<screenshot>https://openmsx.org/images/screenshots/ide.png</screenshot>
+		<screenshot>https://openmsx.org/images/screenshots/tb-underwater.png</screenshot>
+	</screenshots>
+	<developer_name>openMSX team</developer_name>
+	<content_rating type="oars-1.1"/>
+	<update_contact>noreply@openmsx.org</update_contact>
+	<recommends>
+		<control>keyboard</control>
+		<control>pointing</control>
+		<control>gameplad</control>
```

Comment:
Typo here. Not sure whether it's verified by the appstream-glib tool.

Reply:
Thanks, updated. I had checked with the appstream tool, but it did not complain about this.